src/reconcile/rec_write.c

```diff
@@ -2476,9 +2476,9 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
                              * The exception is root pages are never tracked or free'd, they are
                              * checkpoints, and must be explicitly dropped.
                              */
-        if (!__wt_ref_is_root(ref))
-            WT_RET(__wt_btree_block_free(session, mod->mod_replace.addr, mod->mod_replace.size));
-
+        if (!__wt_ref_is_root(ref)) {
```

Comment:
no need for branckets

Reply:
removed here [1c4f7bb](https://github.com/wiredtiger/wiredtiger/pull/10686/commits/1c4f7bb287446e1166c6d19da2501a9c365bff50)